whichone/static/js/controller_module.js

```diff
@@ -0,0 +1,39 @@
+function Controller(input, output) {
```

Comment:
What is output? Pure functions are generally preferable for functional code (I guess you have this for uniformity and as a mechanism to pass state?)

For functional code you want to inject the minimal arguments required for the function. Rather than building in the assumption of whatever `args`/`input` is you'd be better passing in document, game, view, data separately.

Reply:
The controller class doesn't actually output anything so that can be removed